csrc/ir_internal_nodes.h

```diff
@@ -1857,20 +1866,34 @@ class TORCH_CUDA_CU_API TensorDomain : public Val {
     return no_bcast_domain_;
   }
 
-  const std::vector<IterDomain*>& getRootDomain() const {
+  const std::vector<IterDomain*>& root() const {
     return root_domain_;
   };
 
-  const std::vector<IterDomain*>& getRFactorDomain() const {
+  const std::vector<IterDomain*>& rfactor() const {
     return rfactor_domain_;
   };
 
+  const std::vector<IterDomain*>& allocation() const {
+    return allocation_domain_;
+  }
+
+  const std::vector<IterDomain*>& leaf() const {
+    return leaf_domain_;
+  }
+
   // If rfactor domain exists in domain() return it, otherwise return root
   // domain.
   const std::vector<IterDomain*>& getMaybeRFactorDomain() const {
-    return hasRFactor() ? getRFactorDomain() : getRootDomain();
+    return hasRFactor() ? rfactor() : root();
```

Comment:
If we are changing `getRootDomain` to `root`, why not change this to `maybeRfactor`?

Reply:
I agree, let me change it.

Reply:
changed